kroxylicious-schema-validation/src/main/java/io/kroxylicious/proxy/filter/schema/SchemaValidationFilterContributor.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.filter.schema;
+
+import io.kroxylicious.proxy.filter.FilterContributor;
+import io.kroxylicious.proxy.filter.KrpcFilter;
+import io.kroxylicious.proxy.filter.schema.validation.ProduceRequestValidator;
+import io.kroxylicious.proxy.service.BaseContributor;
+
+import static io.kroxylicious.proxy.filter.schema.JsonValueValidator.isValidJson;
+
+public class SchemaValidationFilterContributor extends BaseContributor<KrpcFilter> implements FilterContributor {
```

Comment:
I think there's an important distinction we should make between something being well-formed according to whatever _format_ it uses (e.g. JSON, Avro, Protobuf, XML etc), and valid according to some higher-level schema constraints on top of that format (e.g. the JSON is valid according to some topic specific JSONSchema, or the XML is valid according to a DTD, or an XMLSchema). 

If you accept that distinction as meaningful, then this should be called something like `FormatValidationFilterContributor` (if it should be generic over multiple formats, distinguished only by how it's configured), or possibly `WellFormedJsonValidationFilterContribtor` (if it should be JSON specific)